### DISCUSSION TITLE
Set repository browser via API

### DISCUSSION THREAD

**Discussion by rffontenelle** [WeblateOrg/weblate#10639] [Q&A]
I've read the [REST API](https://docs.weblate.org/en/latest/api.html) doc and haven't found a way to set a component's [repository browser](https://docs.weblate.org/en/latest/admin/projects.html#repository-browser) ('repoweb', as identified in the REST API).

I would like to set programmatically the repoweb for a bunch of components. Is it possible?

**Comment by nijel** [comment]
It will work on PATCH/PUT on the component: https://docs.weblate.org/en/latest/api.html#patch--api-components-(string-project)-(string-component)-
